dokument/infrastructure/src/main/kotlin/no/nav/su/se/bakover/dokument/infrastructure/database/DokumentPostgresRepo.kt

```diff
@@ -198,17 +200,18 @@ class DokumentPostgresRepo(
     /**
      * Henter max antall dokumenter basert på [antallSomSkalHentes]
      */
-    override fun hentDokumenterForDistribusjon(antallSomSkalHentes: Int): List<Dokumentdistribusjon> {
+    override fun hentDokumenterForDistribusjon(antallSomSkalHentes: Int): List<Pair<Dokumentdistribusjon, Distribueringsadresse?>> {
         return dbMetrics.timeQuery("hentDokumenterForDistribusjon") {
             sessionFactory.withSession { session ->
                 """
-                select * from dokument_distribusjon
+                select dd.*, d.distribueringsadresse from dokument_distribusjon dd join dokument d on dd.dokumentid = d.id
```

Comment:
Trenger ikke gjøre denne joinen siden vi henter dokumentet i it.toDokumentdistribusjon(session)

Reply:
Trenger da heller ikke returnere pair